examples/basecoin/x/cool/errors.go

```diff
@@ -0,0 +1,15 @@
+package cool
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	// Cool module reserves error 300-399 lawl
+	CodeIncorrectCoolAnswer sdk.CodeType = 300
+)
+
+// ErrIncorrectCoolAnswer - Error returned upon an incorrect guess
+func ErrIncorrectCoolAnswer(answer string) sdk.Error {
+	return sdk.NewError(CodeIncorrectCoolAnswer, "Incorrect Cool answer - `"+answer+"'")
```

Comment:
use `fmt.Sprintf` here - also I don't think we need quotes around the provided answer

Reply:
resolved